src/www/ui/api/documentation/openapiv2.yaml

```diff
@@ -2482,7 +2482,7 @@ paths:
                   - $ref: '#/components/schemas/User'
                   - type: object
                     properties:
-                      default_group:
+                      defaultGroupName:
```

Comment:
Would rather recommend refactoring `defaultGroup` in `#/components/schemas/User` as a string field which holds the group name rather than group id as there are no other endpoints which uses group ids. And perhaps, move `/groups/{id}` and endpoints under it to use the group names instead.

If you agree, we can document the changes in `/groups/{id}` as part of #2709.
/cc @dushimsam

Reply:
Agreed, I'll push the changes soon.